### PR TITLE
Add missing tryfrom example

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
r? @rust-lang/docs

### CODE REVIEW

File: src/libcore/convert.rs [hunk 1]
```diff
@@ -410,6 +410,24 @@ pub trait TryInto<T>: Sized {
 /// When the `!` type is stablized `Infallible` and `!` will be
 /// equivalent.
 ///
+/// It can be implemented as follows:
```
Comment (Centril) [inline src/libcore/convert.rs]:
```suggestion
/// `TryFrom<T>` can be implemented as follows:
```

**Comment by GuillaumeGomez** [comment]
Updated!

**Pull request merged**